delft/eris.nix

```diff
@@ -590,6 +590,28 @@ in
       (import ../channels.nix).channels;
   };
 
+  # pull nixos metrics from github:NixOS/nixos-metrics to local VictoriaMetrics
+  # every day at 09.00
+  services.cron = {
```

Comment:
This should be a systemd timer. Cron is a pain in the ass to use/test.

Reply:
Fixed.